content/master/concepts/composition-revisions.md

```diff
@@ -14,7 +14,7 @@ what managed resources Crossplane will create in response. Let's say for example
 that you define a `PlatformDB` XR, which represents your organisation's common
 database configuration of an Azure MySQL Server and a few firewall rules. The
 `Composition` contains the 'base' configuration for the MySQL server and the
-firewall rules that is extended by the configuration for the `PlatformDB`.
+firewall rules that are extended by the configuration for the `PlatformDB`.
```

Comment:
There is a 2nd commit in this PR that basically takes a commit that was merged earlier today (https://github.com/crossplane/docs/commit/2ec478c95bb184983eff78323933eaba4a4b3eb4) and recommits it with you as the committer - can you clean this up from this history?